frontend/src/views/Home.vue

```diff
@@ -80,35 +81,58 @@
         <img height="250px" src="@/assets/banner-4.jpg" style="padding-left: 20px; padding-right: 20px;">
       </InfiniteSlideshow>
     </div>
-
+    <!-- Resources -->
     <v-container>
       <HeaderTitle :title="'resources'" />
       <Preview :items="resourceItems"/>
     </v-container>
 
+    <!-- Join Us -->
     <v-container ref="joinus">
       <CommunityLinks ></CommunityLinks>
     </v-container>
 
-    <!-- Support CSESoc -->
-    <v-container id=sponsor >
-      <HeaderTitle :title="'support csesoc'" />
-      <p>
-        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
-        invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam
-        et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
-        </p>
-        <RouterLink to="/sponsors" class="link" >
-          <a
-            style="float:right"
-            target="_blank"
-            v-ripple
-            class="button">
-            Sponsor Us
-          </a>
-        </RouterLink>
-    </v-container>
-
+    <!-- Sponsor -->
+    <div class=sponsor >
+      <v-container>
+        <!-- "support csesoc" title -->
+        <h1 class=sponsor--h1>SUPPORT CSESOC</h1>
+        <!-- row for trophy image and text adjacent -->
+        <v-row no-gutters>
+          <!-- trophy image -->
+          <v-col cols="4" align="center" class="pa-0">
+            <img src="@/assets/trophy.svg" class="sponsor--img">
+          </v-col>
+          <!-- h4 and p text adjacent to trophy image -->
+          <v-col cols="8" class="pa-0">
+            <h4 class=sponsor--h4>Reach Australia's Best Computing Graduates</h4>
+            <p class=sponsor--p> Many of our past members have gone on to work at our previous
+              sponsor companies including Google, Facebook, Jane St and Commonwealth Bank. </p>
+          </v-col>
+        </v-row>
+        <!-- row for network/share image and text adjacent -->
+        <v-row no-gutters class="pa-0">
+          <!-- network image -->
+          <v-col cols="4" align="center" justify="center" class="pa-0">
+            <img src="@/assets/share.svg" class="sponsor--img">
+          </v-col>
+          <!-- h4 and p text adjacent to network image -->
+          <v-col cols="8" class="pa-0">
+            <h4 class=sponsor--h4>Interact With Our Huge Active Community</h4>
+            <p class=sponsor--p> We have an extremely active community of ~3000 CSE students,
+              achieved through our offering of career, social and educational events. </p>
+          </v-col>
```

Comment:
_nit_: This can be done in a for loop for ease of use but I can refactor that. For extra guidance, look at Jie's footer and how he has taken the data away from the html and used vue's for loop to render the data.